common-libs/common/src/shared/seedTools.ts

```diff
@@ -10,74 +10,96 @@ export type KeyOptions = {
   keyTypes: KeyAlgorithmType[]
 }
 
+export type ParseDecryptedSeedResult = {
+  seed: Buffer
+  didMethod: string
+  seedHexWithMethod: string
+  fullSeedHex: string
+  externalKeys?: any[]
+  metadata?: Record<string, any>
+}
+
+const ADDITIONAL_DATA_SEPARATOR = '++;additionalData:'
```

Comment:
Let's change it to `++;additionalDataJson:`, just to make it more future-proof in case we'll ever want to change its format.